docs/src/modeler/network_matrices.jl

```diff
@@ -0,0 +1,14 @@
+# ```@meta
+# CurrentModule = PowerSystems
+# ```
+
+# # Network Matrices
+
+# Use this matrices wisely
```

Comment:
```suggestion
# Use these matrices wisely
```
However, this statement doesn't appear to add much value and you could probably delete it.  Is there something you specifically want to warn users about?

Reply:
The docstrings in the code misspell `Float64` as `FLoat64`.